generate.py

```diff
@@ -113,9 +122,8 @@ def decode_n_tokens(
 
     return new_tokens, new_probs
 
-
 def model_forward(model, x, input_pos):
```

Comment:
We can revert this to original - we don't only need the GIST mask. After that, the KV cache is just the GIST tokens with no mask.